app/controllers/items_controller.rb

```diff
@@ -1,26 +1,9 @@
 class ItemsController < ApplicationController
-  def index
```

Comment:
Pulling this out to the admin controller, good, good. 

![image](https://cloud.githubusercontent.com/assets/3011748/5891952/e078fa38-a46b-11e4-9f43-4bb6b342b695.png)